sunpy/map/frames_test.py

```diff
@@ -0,0 +1,36 @@
+import astropy.units as u
```

Comment:
Things to do:

1. This code should go into https://github.com/sunpy/sunpy/blob/main/sunpy/coordinates/tests/test_frames.py not a new file
2. You do not need most of these imports at all and should be removed.
3. The test should be something like:
```python

def test_angular_radius():
    coord = frames.Helioprojective(0*u.deg,0*u.deg,5*u.km,obstime="2010/01/01T00:00:00", observer="earth")    
    sc = SkyCoord(0*u.deg, 0*u.deg, 5*u.km,obstime="2010/01/01T00:00:00", observer="earth", frame="helioprojective")
    assert coord.angular_radius==sc.angular_radius
```